developer/metadata.md

```diff
@@ -2,18 +2,15 @@
 title:
     - Alfred Api 
     - Developer Guide
-title-slogan:
-    - TODO: THIS SHOULD NOT BE HERE
 date: 01/11/2020
-document-reference: Api/User Guide/2.7.0
```

Comment:
I really think "User guide" is appropriate. "Developer" is an overloaded term, since you don't know if it means "Contributor" or "User" here.